### PR TITLE
add custom esp32s2 board

### PR DESCRIPTION
**Pull request by CircuitART**

### CODE REVIEW

File: ports/espressif/boards/hexky_s2/board.h [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#define DISPLAY_VSCSAD        140
+
+#define DISPLAY_TITLE         "HexKy TFT"
+
+//--------------------------------------------------------------------+
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID           0x303A
+#define USB_PID           0x80DA
```
Comment (hathach) [inline ports/espressif/boards/hexky_s2/board.h:93]: would you mind giving the link to where the PID is allocated for this board.

Comment (CircuitART) [reply ports/espressif/boards/hexky_s2/board.h:93]: here you go, entry 225 & 226
https://github.com/espressif/usb-pids/blob/main/allocated-pids.txt

**Review by hathach — COMMENTED** [review]
Look good, you should also add ci for this new board as well for generating binaries in release tag https://github.com/adafruit/tinyuf2/blob/master/.github/workflows/build_esp32.yml#L22

**Comment by CircuitART** [comment]
add it

**Review by hathach — APPROVED** [review]
perfect, thank you for your PR

**Comment by CircuitART** [comment]
Thank you :)

**Pull request merged**